monitoring/local_monitoring/ReadMe.md

```diff
@@ -0,0 +1,68 @@
+# Local Monitoring with Kruize-Autotune
+
+With Kruize-Autotune's local monitoring mode, we'll explore a demonstration of local monitoring on OpenShift, highlighting how it provides customized recommendations for various load scenarios.
+
+## Getting Started with the Demo
+
+To begin exploring local monitoring capabilities, follow these steps:
+
+### Run the Demo
+
+##### Clone the demo repository:
+```sh
+git clone git@github.com:kruize/kruize-demos.git
+```
+##### Change directory to the local monitoring demo:
+```sh
+cd kruize-demos/monitoring/local_monitoring
+```
+##### Execute the demo script in openshift as: 
+```sh
+./local_monitoring_demo.sh -c openshift
```

Comment:
Also update that the demo can be run on minikube as well with usage options

Reply:
Didn't include as the minikube set-up doesn't run as expected for local monitoring.